### PR TITLE
Add github issue template

### PR DESCRIPTION
**Pull request by has2k1**
#### Problem Description
*Describe the issue and explain **why** it is a problem or what the expected output should be.*

#### Minimal Reproducible Example (reprex)
*The goal of a minimal reprex is to make it as easy as possible to recreate your problem.
If you've never heard of a reprex before, start by reading "What is a reprex",
and follow the advice further down the page.*

```R

```

#### Before you submit
- [What is a reprex?](https://github.com/jennybc/reprex#what-is-a-reprex)
- [How to create a minimal reprex.](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#issues)
- Please do not include session info unless it's explicitly asked for, or you've used `reprex::reprex(..., si = TRUE)` to hide it away.
- If you have a question, ask at the [user mailing list](https://groups.google.com/forum/#!forum/ggplot2) or the [developer mailing list](https://groups.google.com/forum/#!forum/ggplot2-dev).
- Remove this section and remove the section titles and the *hints* below the titles.

### CODE REVIEW

**Comment by hadley** [comment]
Here's my github saved reply for a reprex. Could you please use that?

> Can you please provide a minimal reprex (reproducible example)? The goal of a reprex is to make it as easy as possible for me to recreate your problem so that I can fix it: please [help me help you](https://nypdecider.files.wordpress.com/2014/08/help-me-help-you.gif)! 
>
> If you've never heard of a reprex before, start by reading "[What is a reprex](https://github.com/jennybc/reprex#what-is-a-reprex)", and follow the advice further down the page.

And here's what I say about package version/session info:

> (Please don't include session info unless it's explicitly asked for, or you've used `reprex::reprex(..., si = TRUE)` to hide it away. I've deleted it from your issue to avoid cluttering up the discussion)

(So please remove the section encouraging it and a brief note discouraging it)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[Describe the issue and explain **why** it is a problem or what the expected output should be]
+
+#### Minimal Reproducible Example
+[Make it simple, copy-pastable and include any library calls]
+
+```R
+
+```
+
+[Include the Image(s) if any, of the Minimal Reproducible Example]
```
Comment (hadley) [inline .github/ISSUE_TEMPLATE.md]: I think you can delete both of these statement in favour of recommending the reprex package

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+[This is the output of `packageVersion('ggplot2')`, make sure you have the latest version installed.
+ If comparing against an older version, mention that in the problem description]
+
+#### Remove any sections that do not apply
+[Remove this section too and the one below and remove all the hints in the square brackets]
+
+#### Before you submit
+- [How to create a Minimal Reproducible Example?](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#issues)
+- If you have a question, ask at the [user mailing list](https://groups.google.com/forum/#!forum/ggplot2) or the [developer mailing list](https://groups.google.com/forum/#!forum/ggplot2-dev).
+- [Think about a pull request](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#pull-requests).
```
Comment (hadley) [inline .github/ISSUE_TEMPLATE.md]: I think you should remove this. I'd prefer not to encourage pull requests until I've had a chance to say whether it's likely to get accepted or not.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```R
+
+```
+
+#### Before you submit
+- [What is a reprex?](https://github.com/jennybc/reprex#what-is-a-reprex)
+- [How to create a minimal reprex.](https://github.com/tidyverse/ggplot2/blob/master/CONTRIBUTING.md#issues)
+- Please do not include session info unless it's explicitly asked for, or you've used `reprex::reprex(..., si = TRUE)` to hide it away.
+- If you have a question, ask at the [user mailing list](https://groups.google.com/forum/#!forum/ggplot2) or the [developer mailing list](https://groups.google.com/forum/#!forum/ggplot2-dev).
+- Remove this section and remove the section titles and the *hints* below the titles.
```
Comment (has2k1) [inline .github/ISSUE_TEMPLATE.md]: If the instructions are clear, I think the section demarcations are just extra fluff.

**Comment by hadley** [comment]
Thanks - I'll tweak it a bit myself, and we'll see how it goes. If it works well, I'll add to usesthis as `use_issue_template()`

**Pull request merged**